aikau/src/main/resources/alfresco/layout/css/StickyPanel.css

```diff
@@ -0,0 +1,68 @@
+.alfresco-layout-StickyPanel {
+   bottom: 0;
+   left: 50%;
+   position: fixed;
+   right: 0;
+   &__panel {
+      background: #fff;
```

Comment:
LESS variable? I think we have a primary background colour one that could be used as a seed for a stick panel one. I'm not hugely fussed about this for this first cut as I know we're going to be iterating on it.

I'll probably raise another feature request to LESS-ify the whole thing - so no need to make changes, just wanted to note it.